### ISSUE TITLE
alink中webui咨询

### ISSUE THREAD

**Issue by StaveZhao** [alibaba/Alink#204]
webui 例子中的项目启动之后，拖好流程，运行的时候一直报错，alink已经通过flink部署，是和flink版本有关系吗？？还是需要什么别的配置？？？flink版本1.10 
<img width="1174" alt="image" src="https://user-images.githubusercontent.com/51053710/158580173-2fce8622-f5ca-40d1-bdd1-f4cbc8e3c6c8.png">

**Comment by KangTomwk** [comment]
@StaveZhao  为什么我在alink webui上面拖动某个算子会出现“实验数据建立中，请稍后”，你是怎么拖动流程的呀

**Comment by StaveZhao** [comment]
> 

我就是拖了最简单的流程做测试，没有碰见你说的这个问题

**Comment by KangTomwk** [comment]
@StaveZhao 你是在linux环境打开的webui吗，我用linux部署的，webui会去请求localhost:8080,才导致那个报错的吧

**Comment by StaveZhao** [comment]
> @StaveZhao 你是在linux环境打开的webui吗，我用linux部署的，webui会去请求localhost:8080,才导致那个报错的吧
我翻墙发现的一个博客还没试，看着好像是对的，你可以看看
https://miaowenting.site/2022/01/30/Alink%20%E5%BF%AB%E9%80%9F%E4%BD%BF%E7%94%A8%20WebUI/

**Comment by KangTomwk** [comment]
@StaveZhao
https://miaowenting.site/2022/01/30/Alink%20%E5%BF%AB%E9%80%9F%E4%BD%BF%E7%94%A8%20WebUI/

就是按照这个链接安装的，就是出来的ip都是localhost，不是具体的ip，导致那个报错的

**Comment by Fanoid** [comment]
> webui 例子中的项目启动之后，拖好流程，运行的时候一直报错，alink已经通过flink部署，是和flink版本有关系吗？？还是需要什么别的配置？？？flink版本1.10 <img alt="image" width="1174" src="https://user-images.githubusercontent.com/51053710/158580173-2fce8622-f5ca-40d1-bdd1-f4cbc8e3c6c8.png">

Server 里默认用的是 1.9 的 Flink，你可能需要修改依赖后重新打包。

**Comment by KangTomwk** [comment]
@ Fanoid 你知道web中类似于下面案例中，colNames，source，label ，userParams 这四个参数在哪里标识吗，当前web页面没有这几个赋值的地方？？？当前只有算子及对应参数设置地方
import json
source = RandomTableSourceBatchOp() \
    .setNumRows(100) \
    .setNumCols(10)
colNames = source.getColNames()
source = source.select("*, case when RAND() > 0.5 then 1. else 0. end as label")
label = "label"
userParams = {
    'featureCols': json.dumps(colNames),
    'labelCol': label,
    'batch_size': 16,
    'num_epochs': 1
}
tfTableModelTrainBatchOp = TFTableModelTrainBatchOp() \
    .setUserFiles(["https://alink-release.oss-cn-beijing.aliyuncs.com/data-files/tf_dnn_train.py"]) \
    .setMainScriptFile("https://alink-release.oss-cn-beijing.aliyuncs.com/data-files/tf_dnn_train.py") \
    .setUserParams(json.dumps(userParams)) \
    .linkFrom(source)
tfTableModelTrainBatchOp.print()
